combat-trainer.lic

```diff
@@ -352,6 +355,12 @@ class LootProcess
     end
   end
 
+  def should_perform_ritual(game_state)
+    unless game_state.prepare_consume
+      (DRSkill.getxp('Thanatology') <= 30 || (@necro_train_first_aid && DRSkill.getxp('First Aid') <= 30)) || !@skin
+    end
```

Comment:
Man the logic here is becoming a trainwreck. I'm not even sure why we care if they're skinning. That will happen anyways. I vote we raise the checks to 32 instead of 30 and just ignore the whole skin requirement. That leaves us with a much saner.

```
  def should_perform_ritual(game_state)
    !game_state.prepare_consume &&
    !(@necro_train_first_aid && DRSkill.getxp('First Aid') <= 32)) &&
    DRSkill.getxp('Thanatology') <= 32
  end
```

Now it scans much better as You should perform the ritual if you're not preparing consume, and your not trying to train FA while it's not locked and you're not thanatology locked.